examples/DataStax/basic/basic-auto-paged.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
+const cassandra = require("scylladb-javascript-driver");
+const { getClientArgs } = require("../util");
+const async = require("async");
+const _Row = require("../../../lib/types/row");
+const _ResultSet = require("../../../lib/types/result-set");
+
+const client = new cassandra.Client(getClientArgs());
+
+/**
+ * Example using async library for avoiding nested callbacks
+ * See https://github.com/caolan/async
+ * Alternately you can use the Promise-based API.
+ *
+ * Inserts 100 rows and retrieves them with paging queries
+ * with automatic paging paging
+ */
+
+async.series(
+    [
+        function connect(next) {
+            client.connect(next);
+        },
+        function dropKeyspace(next) {
+            const query = "DROP KEYSPACE IF EXISTS examples";
+            client.execute(query, next);
+        },
+        function createKeyspace(next) {
+            const query =
+                "CREATE KEYSPACE IF NOT EXISTS examples WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1' }";
```

Comment:
:wrench: `SimpleStrategy` is deprecated. Please use `NetworkTopologyStrategy` instead, as a drop-in replacement. Ref #179.